packages/app/src/cli/models/extensions/specifications/flow_trigger.ts

```diff
@@ -30,11 +42,16 @@ const flowTriggerSpecification = createExtensionSpecification({
   schema: FlowTriggerExtensionSchema,
   singleEntryPath: false,
   appModuleFeatures: (_) => [],
```

Comment:
hey @MaxLeduc , since this extension has a config file, we need to add 

```javascript
appModuleFeatures: (_) => ['bundling'],
```

for it to upload the configuration data to app_extensions_config table.

Reply:
This is related https://github.com/Shopify/partners-web-platform/pull/1265. We need to publish the configuration before disabling the form in partners dahsboard

Reply:
Got it!

Reply:
wait, the 'bundling' feature is to include output files in the `bundle.zip`. This is where we include the compiled javascript of UI Extensions for instance. It shouldn't apply for flow, since we only have config files

Reply:
it's true that we only have config files for flow. We need to use the bunlding feature to update the extension's config since the initial registration call sends in a empty config object

- Config is hardcoded to empty [here](https://github.com/Shopify/cli/blob/3f7fbad7db0c6e28c10a70a550d5a859cd5eddc7/packages/app/src/cli/services/dev/create-extension.ts#L36)
	- Code path deploy.ts -> [`*ensureDeployContext*`](https://github.com/Shopify/cli/blob/cedea083cccbd10f0cf0ca4d09a7a8e7dfb2c34b/packages/app/src/cli/services/context.ts#L291) ->[`ensureDeploymentIsPresence`](https://github.com/Shopify/cli/blob/3f7fbad7db0c6e28c10a70a550d5a859cd5eddc7/packages/app/src/cli/services/context/identifiers.ts#L38) -> [`ensureExtensionsIds`](https://github.com/Shopify/cli/blob/647bdcfd9f0f8355f01fbb1aaac044fbbc4ca905/packages/app/src/cli/services/context/identifiers-extensions.ts#L17) -> [`createExtensions`](https://github.com/Shopify/cli/blob/647bdcfd9f0f8355f01fbb1aaac044fbbc4ca905/packages/app/src/cli/services/context/identifiers-extensions.ts#L85) which creates the extension without the configurations.
- In this [deploy task](https://github.com/Shopify/cli/blob/d925436d2c0698334c6d7fedd0bb361cc7f5291a/packages/app/src/cli/services/deploy.ts#L82-L83), we gather all the [extension configurations](https://github.com/Shopify/cli/blob/d925436d2c0698334c6d7fedd0bb361cc7f5291a/packages/app/src/cli/services/deploy.ts#L84-L88). However, without being a bundle, theme, or functions, the deploy process doesn't do anything with the config data.
- That's why I'd like to hook it up into the `bundle` process.

Reply:
Alternatively, we can change the hard-coded empty config object to use the extensions' `deployConfig`

Reply:
[the issue](https://github.com/Shopify/flow/issues/19464) i picked up is for when unifed_deployment is disabled 😆. In that case, @ryanische do you think the issue is still relevant?

Reply:
Is there a way we can resolve this in a subsequent PR? I'd be down to remove `'bundling'` and test that separately so it won't be a blocker for this PR which needs to go in for convergence.